ts/screens/wallet/AddCardScreen.tsx

```diff
@@ -78,7 +113,7 @@ export default class AddCardScreen extends React.Component<Props, State> {
     const primaryButtonProps = {
       block: true,
       primary: true,
-      onPress: () => this.props.navigation.navigate(ROUTES.WALLET_HOME),
+      onPress: this.submit,
```

Comment:
this button should be enabled only when the form data is valid

Reply:
Considering that that is mostly a UI-only behavior, and considering that this PR is necessary to add new payment methods to one's wallet (which is necessary to test the payment process), I would recommend we create an appropriate story to implement this behavior and move on with this PR so that we can allow a new user to add payment methods and then make a payment

Reply:
I'm not in agreement but yes, please create a story for that and go head